tests/performance_tests/performance_test_basic.py

```diff
@@ -94,17 +94,20 @@ def argsStr(self) -> str:
         maximumClients: int = 0
         loggingDict: dict = field(default_factory=lambda: { "bios": "off" })
         prodsEnableTraceApi: bool = False
+        isNodeosv2: bool = False
```

Comment:
Still wondering if this should be a `NodeosVersion` enumeration that would allow extending it, say if some functionality changes between `v3` and `v4` instead of requiring adding another boolean flag to be passed around.